### PR TITLE
WhiteNoise Core Evaluation

### LINKED ISSUE #179: Adding tests for WhiteNoise-Core
Adding stochastic evaluation of WhiteNoise-Core (Yarrow)

### PR DESCRIPTION
**Pull request by ankit-oss**
Adding DP test and Bias test for WhiteNoise-Core aggregates like MEAN, VAR, MOMENT, COVARIANCE

Closes #179

### CODE REVIEW

File: service/datasets/evaluation/PUMS_1000.csv [hunk 1]
```diff
@@ -0,0 +1,1001 @@
+age,sex,educ,race,income,married
```
Comment (eedeleon) [inline service/datasets/evaluation/PUMS_1000.csv:1]: We should not upload the dataset. Can we download it in tests/conftest.py and add a gitignore for *.csv,

File: tests/evaluation/test_yarrow.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import logging
```
Comment (eedeleon) [inline tests/evaluation/test_yarrow.py:1]: we should remove yarrow and make it test whitenoise_core

File: evaluation/WhiteNoise-Core Evaluation.ipynb [hunk 3]
```diff
@@ -0,0 +1,327 @@
+{
+ "cells": [
```
Comment (eedeleon) [inline evaluation/WhiteNoise-Core Evaluation.ipynb:2]: can we take the space out of the name. We should find a convention but spaces are hard

Comment (ankit-oss) [reply evaluation/WhiteNoise-Core Evaluation.ipynb]: Sure. Fixed the name removing space

File: tests/evaluation/test_yarrow.py (hunk 2 above)
Comment (ankit-oss) [reply tests/evaluation/test_yarrow.py]: Updated yarrow to whitenoise-core in all the files.

File: service/datasets/evaluation/PUMS_1000.csv (hunk 1 above)
Comment (ankit-oss) [reply service/datasets/evaluation/PUMS_1000.csv]: Downloading via conftest.py. Updated .gitignore with the entry for the PUMS_1000.csv like the other CSV files already mentioned in it.

File: tests/evaluation/test_stochastic.py [hunk 4]
```diff
@@ -115,32 +115,4 @@ def test_groupby_avg(self):
         assert(dp_res == True)
         assert(acc_res == True)
```
Comment (eedeleon) [inline tests/evaluation/test_stochastic.py:116]: These checks a can all be:

 assert (acc_res) 

just fyi

Comment (ankit-oss) [reply tests/evaluation/test_stochastic.py]: Done

**Pull request merged**